process/security_baseline.md

```diff
@@ -0,0 +1,253 @@
+|A security policy is published and followed for vulnerability disclosure and response.|To provide clear guidance on how security findings are reported and the expected response.|SECURITY.md is established as part of the [OpenSSF default community health file](https://github.com/ossf/.github). It serves as the security policy, and provides generic information about how vulnerabilities are reported to the project. Project maintainers SHALL follow the [guide to coordinated vulnerability disclosure](https://github.com/ossf/oss-vulnerability-guide/blob/main/maintainer-guide.md#response-process) to provide project-specific vulnerability disclosure and response process. GitHub private vulnerability reporting SHALL be enabled at the repo level.<br /><br />The security policy SHALL be added to SECURITY_INSIGHTS.yml under section “vulnerability-reporting”.<br /><br />Examples SECURITY.md: <br />OpenSSF: [Sigstore](https://github.com/sigstore/cosign?tab=security-ov-file), [Scorecard](https://github.com/ossf/scorecard/blob/49c0eed3a423f00c872b5c3c9f1bbca9e8aae799/SECURITY.md)<br /><br />Example SECURITY_INSIGHTS.yml: <br />OpenSSF: [GUAC](https://github.com/guacsec/guac/blob/main/SECURITY-INSIGHTS.yml) <br />CNCF: [capsule](https://github.com/projectcapsule/capsule/blob/main/SECURITY-INSIGHTS.yml), [Kyverno](https://github.com/kyverno/kyverno/blob/main/SECURITY-INSIGHTS.yml)|[Scorecard checks security policy](https://github.com/ossf/scorecard/blob/7ce8609469289d5f3b1bf5ee3122f42b4e3054fb/docs/checks.md#security-policy) existence in a project and reports findings.<br /><br />The SECURITY_INSIGHTS.yml file identifies the security policy file (typically SECURITY.md).<br /><br />Example security insights :<br />OpenSSF: [GUAC](https://github.com/guacsec/guac/blob/main/SECURITY-INSIGHTS.yml)|
+|Data in transit must be protected by cryptographic means.|To protect data from unauthorized access,  ensure data integrity, confidentiality and availability.|[Encrypting Data in Transit and at Rest](#Encrypting-Data-in-Transit-and-at-Rest) provides detailed rationale behind the baseline and implementation guidance.|[Use curl and openssl to verify](https://docs.google.com/document/d/1-NBXdKvEJ9Wsh2i7lDNYven4fY9Bn6uvNJM5ySlMrdg/edit#heading=h.59ofury7nggp) the TLS protocol version, cipher and certificate chain for HTTPS traffic.|
```

Comment:
We had an issue with this in this during the prometheus security review in CNCF. I think the question here is depending on the model... E.g. if you run a service on istio, you would have TLS, but you wouldn't want the servers to actually run TLS themselves too... This is a weird one because its subjective on deployment and threat model. I would suggest against making this a requirement.

Reply:
+1